vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -41,8 +42,11 @@ var (
 		tlsPrivateKey: flag.String("tls-private-key", "/etc/tls-certs/serverKey.pem", "Path to server certificate key PEM file."),
 	}
 
-	address   = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
-	namespace = os.Getenv("NAMESPACE")
+	address         = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
+	namespace       = os.Getenv("NAMESPACE")
+	webhookHostName = flag.String("webhookHostName", "", "path for admission controller.Blank for default")
```

Comment:
Can you change the description to "Address under which webhook is registered. Used when registerByURL is set to true."